distribution/resources/bin/gateway

```diff
@@ -192,6 +192,8 @@ java_cmd_gateway () {
     $JAVA_OPTS \
     -Dmgw-runtime.home=${GW_HOME} \
     -Dballerina.home=${GW_HOME}/runtime \
+    -Djava.util.logging.config.class="org.ballerinalang.logging.util.LogConfigReader" \
```

Comment:
Is this something we forgot to add or something fixing an issue?

Reply:
Actually, these variables are only required when observability is enabled in the gateway. Is it required to create another issue for this?

Reply:
No, its fine. Just wanted to know why we needed to add it. It'd be perfect it we could have this info in git commit message. But that will complicated things now 😃